deploy-preview.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+echo -e "\033[0;32mDeploying updates to GitHub...\033[0m"
+
+# Build the project with docker in a sibling repository.
+docker run --rm \
+	-v $(pwd):/src \
+	-v $(pwd)/../preview:/target \
+	--user $(id -u):$(id -g) \
+	klakegg/hugo:0.55.6 \
+	--config config.preview.toml
+
+# Go To Preview folder
+cd ../preview
+# Add changes to git.
```

Comment:
This assumes we are at the root - would it make sense to include finding the directory of the script, and then moving relative to it?

Reply:
Worse yet, it assume you've cloned the preview repo as a sibling to the root.
Current state of this script is more documentation than very usable as-is.